src/services/embedding_service.py

```
@@ -0,0 +1,272 @@
+            return chunk, False
+
+        try:
+            # Create the prompt for generating contextual information
+            prompt = f"""<document>
+{full_document[:25000]}
+</document>
+Here is the chunk we want to situate within the whole document
+<chunk>
+{chunk}
+</chunk>
+Please give a short succinct context to situate this chunk within the overall document for the purposes of improving search retrieval of the chunk. Answer only with the succinct context and nothing else."""
```

Ruff (0.11.9)

Line too long (206 > 120)

(E501)